.eslintrc

```diff
@@ -0,0 +1,20 @@
+---
+extends: react-app
```

Comment:
@smiller171 ok great. I'm going to merge this in shortly and then add a bunch of React specific rules. Unfortunately, some aren't fixable so it will be a royal pain to fix everything, but worth it to promote good practice going forward.